faq/index.md

```diff
@@ -2,153 +2,280 @@
+If you connect to the mesh network indirectly over the internet (for example over a cable ISP connection) this is by [tinc VPN](https://www.tinc-vpn.org/) and is secure.
 
-You can also help your neighborhood by providing access points for local parks, cafes and bars. Local businesses are often keen to host a node once you explain the benefits to them.
 
-## <a name="coverage"></a>Is it possible for the mesh to cover the whole country?
+## <a name="userdata"></a>What is your user data policy?
 
-Yes, there is a mesh network called "Guifi" in Spain that covers a large part of the country. In order to do this, they have a backbone of many long distance WiFi connections and also community owned fiber.
+NYC Mesh does not collect, store, or log any user traffic or content that passes through our network.
 
-## <a name="isp"></a>Are you an Internet Service Provider (ISP)?
+NYC Mesh may periodically look at [traffic headers](https://en.wikipedia.org/wiki/IPv6_packet) for troubleshooting purposes as traffic passes through the NYC Mesh network, however, this data is never logged or stored. NYC Mesh may collect aggregate statistics to monitor traffic flow to ensure a successful network.
 
-No, we are not an ISP, we are a community that shares ownership of a network. Collectively we have our own internet connection at our "supernode". Also some of our members share their own internet connections. 
+The NYC Mesh network is unencrypted, easy to join, and depends on donated internet from individuals and other companies which is outside the control of NYC Mesh, therefore NYC Mesh is unable to detect or prevent traffic monitored at those access points.
 
-Our network peers with other networks at an internet exchange point and provides access to the internet without traditional ISPs. We are a non-profit project of the New York chapter of the Internet Society (ISOC-NY).
+NYC Mesh will comply with all laws in the jurisdictions where it operates; however, as per our community policy, no data is collected or stored, and therefore no data exists to provide to agents who request it.
 
-## <a name="node"></a>What is a node?
+## <a name="ownership"></a>Who owns NYC Mesh?
 
-A node is a location with one or more routers or antennas, i.e. it is usually a dot on our map. Some dots have multiple apartments with installs in the same building so that would be one node per apartment. A supernode will have multiple radios, routers, antennas and maybe a server too.
+We are a non-profit project of the New York chapter of the [Internet Society](https://www.internetsociety.org/) (ISOC-NY). Most of the equipment that makes up the Mesh is owned by the individual members. Some common infrastructure is owned or rented by the project under ISOC-NY, or by individual members who are donating it. Intellectual property is limited and is under open licenses such as GNU and Creative Commons. 
```

Comment:
Consider linking to https://www.gnu.org/licenses/ & https://creativecommons.org/ – these license terms may as well be greek without some context.

Reply:
will do.